packages/contracts-bedrock/contracts/universal/CrossDomainMessenger.sol

```diff
@@ -345,17 +357,22 @@ abstract contract CrossDomainMessenger is
             "CrossDomainMessenger: message has already been relayed"
         );
 
+        // If there is not enough gas left to perform the external call and finish the execution,
+        // silently fail and assign the message to the failedMessages mapping.
```

Comment:
Not a big deal, so just a comment but I think instead of "silently fail", it'd be better verbiage to say something to the tune of "return early" since silently failing implies a resulting unexpected state or state to be out of wack.